kbase-extension/static/kbase/js/widgets/narrative_core/kbaseNarrativeManagePanel.js

```diff
@@ -34,9 +34,9 @@ define([
 ) {
     'use strict';
     return new KBWidget({
-        name: "kbaseNarrativeManagePanel",
+        name: 'kbaseNarrativeManagePanel',
```

Comment:
That was where all the work was done to fix the bug where the wrong narrative was being copied. Also, some minor refactor and cleanup. It should be safe - there have been no complaints from CI or Next.